examples/simple/src/tags/TagEdit.tsx

```diff
@@ -32,7 +32,7 @@ const TagEdit = props => (
                 basePath="/posts"
                 resource="posts"
                 filter={{ tags: [props.id] }}
-                title=" "
+                title={<span />}
```

Comment:
Worth noting @fzaninotto  : The " " caused the document title to get messed up due to the page having two separate titles. Passing through an empty span means the page title and the UI title only shows the `Tags #` title